misc/scripts/install-local.sh

```diff
@@ -372,14 +365,14 @@ function hashcheck() {
 
 fancy_message info "Retrieving packages"
 mkdir -p "$SRCDIR"
-cd "$SRCDIR"
+cd "$SRCDIR" 2> /dev/null || ( fancy_message warn "Couldn't enter ${SRCDIR}" && exit 1)
```

Comment:
no smasher thingys. It should be `could not`

Reply:
They are called contractions

Reply:
Fixed in https://github.com/pacstall/pacstall/pull/276/commits/0691fb5e7635a168f4f677bc139b3ec1a9983765